taichi/common/core.h

```diff
@@ -141,7 +141,7 @@ static_assert(__cplusplus >= 201402L, "C++14 required.");
 #endif
 
 #define TI_STATIC_ASSERT(x) static_assert((x), #x);
-#define TI_ASSERT(x) TI_ASSERT_INFO((x), #x)
+#define TI_ASSERT(x) TI_ASSERT_INFO((x), "Assertion failure: " #x)
```

Comment:
IMO this is more friendly than simply `#x`, e.g.:
```cpp
TI_ASSERT(p != nullptr);
```

Before:
```
RuntimeError: [expr.c@233] p != nullptr
```
This may make people think it's `p == nullptr` the expected value, while `p != nullptr` cause the error.

After:
```
RuntimeError: [expr.c@233] Assertion failure: p != nullptr
```